src/Parser/Mage/Frost/Modules/Items/LadyVashjsGrasp.js

```diff
@@ -0,0 +1,59 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+const PROC_INTERVAL_MS = 10000;
+
+/**
+ * Lady Vashj's Grasp:
+ * While Icy Veins is active, you gain 1 charge of Fingers of Frost every 10 sec.
+ */
+class LadyVashjsGrasp extends Analyzer {
+
+  static dependencies = {
+		combatants: Combatants,
+	};
+
+  icyVeinsAppliedTimestamp; // the legendary applies its own buff, but it exactly tracks Icy Veins
```

Comment:
It's generally recommended to have an explicit default value of `null` or `undefined`.

Reply:
Oh, like to show to the developer reading my code "yes I know this is undefined" ?